compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -1335,6 +1361,30 @@ void propagate_concat_quantparam(luci::CircleConcatenation *concat, loco::DataTy
  */
 void propagate_pad_v2_quantparam(luci::CirclePadV2 *pad_v2, loco::DataType quant_type)
 {
+  auto const_value_node = dynamic_cast<luci::CircleConst *>(pad_v2->arg(2));
```

Comment:
Why not move this inside the below if statement?

Reply:
Nice catch! This is leftover of attempt to unify this piece with code below =)